dbfit-java/informix/TestDbConnectionDbFitInformixINFORMIX.properties

```diff
@@ -0,0 +1,4 @@
+service=localhost:16466
+username=dbfit
+password=DBFIT
+database=dbfit_ifx
```

Comment:
Right. I'll add the support for the `.custom` properties files for Informix too. They're not there in the `build.gradle` script.